zmon_cli/client.py

```diff
@@ -801,7 +801,25 @@ def get_grafana_dashboard(self, grafana_dashboard_id: str, **kwargs) -> dict:
         """
         current_span = extract_span_from_kwargs(**kwargs)
         current_span.set_tag('grafana_dashboard_id', grafana_dashboard_id)
-        resp = self.session.get(self.endpoint(GRAFANA, grafana_dashboard_id), timeout=self._timeout)
+        resp = self.session.get(self.endpoint(GRAFANA, grafana_dashboard_id, trailing_slash=False), timeout=self._timeout)
+
+        return self.json(resp)
+
+    @trace(pass_span=True)
+    @logged
+    def delete_grafana_dashboard(self, grafana_dashboard_id: str, **kwargs) -> dict:
+        """
+        Delete Grafana dashboard.
+
+        :param grafana_dashboard_id: Grafana dashboard ID.
```

Comment:
I think we need to specify that this is either old grafana dashboard ID, or the new UID eg `EZfmMJIZz`

Reply:
not just here, but in other commands, like get and put